### ISSUE TITLE
How support weights to data points in FFM while training ?

### ISSUE THREAD

**Issue by akshit96** [aksnzhy/xlearn#105]
Is there a way to do that ?

**Comment by akshit96** [comment]
Giving a weight to each row so that it appears along with it's error in the cost function while training and validation.

**Comment by akshit96** [comment]
@aksnzhy Can you guide how should I go about making changes if I have to do it myself?

**Comment by aksnzhy** [comment]
@akshit96 

You can see the code : https://github.com/aksnzhy/xlearn/blob/master/src/score/linear_score.cc

in line 75: 

` w[idx_g] -= (learning_rate_ * gradient);`

Maybe you can change it to 

` w[idx_g] -= (learning_rate_ * gradient * weight);`
